fix(docs): Update `@astrojs/sitemap` readme to clarify build output location (#8844)
withastro/astro: packages/integrations/sitemap/README.md

Note that unlike other configuration options, `site` is set in the root `defineConfig` object, rather than inside the `sitemap()` call.

> Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You should find both `sitemap-index.xml` and `sitemap-0.xml` in your output directory—by default, `/dist`.

Reviewer: ```suggestion
Now, [build your site for production](https://docs.astro.build/en/reference/cli-reference/#astro-build) via the `astro build` command. You will find both `sitemap-index.xml` and `sitemap-0.xml` in the `dist/` folder (or your custom [output directory](https://docs.astro.build/en/reference/configuration-reference/#outdir) if set).
```
What do you think about something like this as being a little smoother, and linking here to the output directory config option? 

I also considered just: 

> in your [output directory](https://docs.astro.build/en/reference/configuration-reference/#outdir).

but I think maybe enough people will just use the default `dist/` folder that mentioning it explicitly is still helpful?
